src/main/c/Include/pyembed.h

```diff
@@ -42,6 +42,27 @@ struct __JepThread {
 };
 typedef struct __JepThread JepThread;
 
+#if defined(Py_LIMITED_API) && Py_LIMITED_API+0 < 0x03090000
+    /*
+     * On versions before 3.8, there was no way to get the current `PyInterpreterState*`
+     * using the Stable ABI.
+     *
+     * Therefore, we simply assume there is one and only one active interpreter (no sub-interpreters)
+     */
+    #define JEP_ASSUME_SINGLE_INTERPRETER
```

Comment:
This is an atrocious hack. Subinterpreters are an essential concept in `jep`.......